test/parallel/test-npm-version.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const path = require('path');
+const exec = require('child_process').exec;
+const assert = require('assert');
+
+const npmPath = path.join(
+  __dirname,
+  '..',
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
+
+exec(`${process.execPath} ${npmPath} -v`, {
```

Comment:
`execSync()`?

Reply:
since this is in parallel I think we should keep it as exec. Thoughts?

Reply:
I don't think it makes any difference in this particular case. Feel free to ignore the nit if you want though.

Reply:
Do we even need to `exec`? We could read the `version` field from npm's `package.json`.